### ISSUE TITLE
accessibility of "toggle switch"

### ISSUE THREAD

**Issue by JoaoEnesGoncalves** [lumada-design/hv-uikit-react#604]
accessibility of "toggle switch"

**Describe the bug**

**To Reproduce**
1. run the cmd 'pa11y --debug --reporter json --standard WCAG2AA --root-element "#root > div.Component-content-4" http://localhost:9002/iframe.html?id=coretoggle--smoke'
2. verify results

**Expected behavior**
- no accessibility issues for accessibility standards WCAG2AA

**Screenshots**

![toggle_defajult](https://user-images.githubusercontent.com/9842927/66328665-f761ff80-e924-11e9-9fb6-ce9870a4f99b.PNG)

**Additional context**
```
{
  "code": "WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputCheckbox.Name",
  "type": "error",
  "typeCode": 1,
  "message": "This checkboxinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.",
  "context": "<input class=\\"
  MuiPrivateSwitchBase - input - 200\\ " id=\\"
  defaultLabels\\ " type=\\"
  checkbox\\ " value=\\"\\
  " checked=\\"\\
  ">",
  "selector": "#defaultLabels",
  "runner": "htmlcs",
  "runnerExtras": {}
},
```

```
 {
  "code": "WCAG2AA.Principle1.Guideline1_3.1_3_1.F68",
  "type": "error",
  "typeCode": 1,
  "message": "This form field should be labelled in some way. Use the label element (either with a or attribute or wrapped around the form field), or title, aria - label or aria - labelledby  attributes as appropriate."",
  "context": "<input class=\\"
  MuiPrivateSwitchBase - input - 200\\ " id=\\"
  defaultLabels\\ " type=\\"
  checkbox\\ " value=\\"\\
  " checked=\\"\\
  ">",
  "selector": "#defaultLabels",
  "runner": "htmlcs",
  "runnerExtras": {}
}

```

```
{
    "code": "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail",
    "type": "error",
    "typeCode": 1,
    "message": "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.71:1. Recommendation:  change text colour to #737373.",
    "context": "<p class=\\"
    Typography - normalText - 174 Typography - baseFontFamily - 186 Typography - margin - 187 Switch - labelDeselected - 155 Switch - labelLeftPositioning - 157\\ " theme=\\" [object Object]\\
    ">Off</p>",
    "selector": "#defaultLabels_leftButton > p",
    "runner": "htmlcs",
    "runnerExtras": {}
  }
```
